README.md

```diff
@@ -1,2 +1,90 @@
 # Capstone-project
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
```

Comment:
- Let's follow given [template](https://github.com/microverseinc/readme-template) and customize your readme file.

Reply:
what should i do over here ? i dont understand

Reply:
Just follow the template and add information related to your project like description, links and command to run the project.